dags/glam_dev.py

```diff
@@ -0,0 +1,445 @@
+    parameters=("submission_date:DATE:{{ds}}",),
+    arguments=("--replace",),
+    dag=dag,
+    docker_image="gcr.io/moz-fx-data-airflow-prod-88e0/glam-dev-bigquery-etl:latest",
+)
+client_scalar_probe_counts = gke_command(
+    task_id="client_scalar_probe_counts",
+    command=[
+        "python3",
+        "script/glam/run_scalar_agg_clustered_query.py",
+        "--submission-date",
+        "{{ds}}",
+        "--dst-table",
+        "clients_scalar_probe_counts_v1",
+        "--project",
+        prod_project_id,
+        "--tmp-project",
+        tmp_project,
+        "--dataset",
+        dev_dataset_id,
+    ],
+    docker_image="gcr.io/moz-fx-data-airflow-prod-88e0/glam-dev-bigquery-etl:latest",
+    dag=dag,
+)
+
+# SubdagOperator uses a SequentialExecutor by default
+# so its tasks will run sequentially.
+# Note: In 2.0, SubDagOperator is changed to use airflow scheduler instead of
+# backfill to schedule tasks in the subdag. User no longer need to specify
+# the executor in SubDagOperator. (We don't but the assumption that Sequential
+# Executor is used is now wrong)
+clients_histogram_bucket_counts = SubDagOperator(
+    subdag=repeated_subdag(
+        GLAM_DAG,
+        "clients_histogram_bucket_counts",
+        default_args,
+        dag.schedule_interval,
+        dev_dataset_id,
+        ("submission_date:DATE:{{ds}}",),
+        25,
+        None,
+    ),
+    task_id="clients_histogram_bucket_counts",
+    dag=dag,
+)
+
+clients_histogram_probe_counts = bigquery_etl_query(
+    task_id="clients_histogram_probe_counts",
+    destination_table="clients_histogram_probe_counts_v1",
+    dataset_id=dev_dataset_id,
+    project_id=prod_project_id,
+    owner="efilho@mozilla.com",
+    date_partition_parameter=None,
+    arguments=("--replace", "--clustering_fields=metric,channel"),
+    dag=dag,
+)
+
+""" No dev extracts or imports in this first phase. 
+We will check the results in the destination tables.
+Besides, there's no dev bucket set up
+
+extract_counts = SubDagOperator(
+    subdag=extract_user_counts(
+        GLAM_DAG,
+        "extract_user_counts",
+        default_args,
+        dag.schedule_interval,
+        dev_dataset_id,
+        "user_counts",
+        "counts"
+    ),
+    task_id="extract_user_counts",
+    dag=dag
+)
+
+extracts_per_channel = SubDagOperator(
+    subdag=extracts_subdag(
+        GLAM_DAG,
+        "extracts",
+        default_args,
+        dag.schedule_interval,
+        dev_dataset_id
+    ),
+    task_id="extracts",
+    dag=dag,
+)
+
+# Move logic from Glam deployment's GKE Cronjob to this dag for better dependency timing
+glam_import_image = 'gcr.io/moz-fx-dataops-images-global/gcp-pipelines/glam/glam-production/glam:2022.03.0-17'
+
+base_docker_args = ['/venv/bin/python', 'manage.py']
+
+env_vars = dict(
+    DATABASE_URL = Variable.get("glam_secret__database_url"),
+    DJANGO_SECRET_KEY = Variable.get("glam_secret__django_secret_key"),
+    DJANGO_CONFIGURATION = "Prod",
```

Comment:
Just making sure, should this be "Prod"?

Reply:
This code is commented out. The export/import steps are not part of this dag just yet, they will be included in the future. I'll still discuss with the team where to send data to, but not to Prod for sure :)